src/Entity/SubscriptionPricing.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PlugAndPay\Sdk\Entity;
+
+class SubscriptionPricing
+{
+    private string $amount;
+    private string $amountWithTax;
+    private array $discounts;
+    private int $quantity;
+    private float $tax;
+    private bool $isTaxIncluded;
+
+    public function amount(): string
+    {
+        return $this->amount;
+    }
+
+    public function setAmount(string $amount): SubscriptionPricing
```

Comment:
De return waardes van de setters worden door phpstorm automatisch de class (`: SubscriptionPricing`), maar eigenlijk is het beter om overal `: self` te gebruiken. Volgens mij doen we dat overal weer anders.

Reply:
Goeie, ik zou alles even controleren in de entiteiten en commit dat wel